ui/src/tempVars/components/TemplateVariableEditor.tsx

```diff
@@ -100,70 +103,83 @@ class TemplateVariableEditor extends PureComponent<Props, State> {
     const TemplateBuilder = this.templateBuilder
 
     return (
-      <div className="edit-temp-var">
-        <div className="edit-temp-var--header">
-          <h1>{isNew ? 'Create' : 'Edit'} Template Variable</h1>
-          <div className="edit-temp-var--header-controls">
+      <OverlayContainer maxWidth={650}>
+        <OverlayHeading title={this.title}>
+          <div className="btn-group--right">
             <button
-              className="btn btn-default"
+              className="btn btn-default btn-sm"
               type="button"
               onClick={onCancel}
             >
               Cancel
             </button>
             <button
-              className="btn btn-success"
+              className="btn btn-success btn-sm"
               type="button"
               onClick={this.handleSave}
               disabled={!this.canSave}
             >
               {this.saveButtonText}
             </button>
           </div>
-        </div>
-        <div className="edit-temp-var--body">
-          <div className="edit-temp-var--body-row">
-            <div className="form-group name">
+        </OverlayHeading>
+        <OverlayBody>
+          <div className="faux-form">
+            <div className="form-group col-sm-6">
               <label>Name</label>
               <input
                 type="text"
-                className="form-control"
+                className="form-control input-sm form-astronaut"
                 value={nextTemplate.tempVar}
                 onChange={this.handleChangeName}
                 onKeyPress={this.handleNameKeyPress}
                 onBlur={this.formatName}
+                spellCheck={false}
               />
             </div>
-            <div className="form-group template-type">
+            <div className="form-group col-sm-6">
               <label>Type</label>
               <Dropdown
                 items={TEMPLATE_TYPES_LIST}
                 onChoose={this.handleChooseType}
                 selected={this.dropdownSelection}
-                buttonSize=""
+                buttonSize="btn-sm"
+                className="dropdown-stretch"
               />
             </div>
-          </div>
-          <div className="edit-temp-var--body-row">
             <TemplateBuilder
               template={nextTemplate}
               source={source}
               onUpdateTemplate={this.handleUpdateTemplate}
             />
+            <div className="form-group text-center form-group-submit col-xs-12">
+              <ConfirmButton
+                text={this.isDeleting ? 'Deleting...' : 'Delete'}
```

Comment:
would you consider changing this to not using a ternary? maybe a getter would be useful here